review/management/commands/pcrbackup.py

```diff
@@ -19,10 +20,14 @@ def handle(self, *args, **kwargs):
         else:
             answer = input('Output directory already exists. Overwrite existing contents? (y/N): ')
             if answer.lower() != 'y':
```

Comment:
This isn't something new so it's prob fine but how come you are allowing 'y' and 'Y'?  Isn't it usually convention to require y to be lowercase and N to be uppercase?

Reply:
I could totally be wrong on this, but I think the `y/N` convention is just conveying that `N` is the default value if you press enter: it's not actually enforcing that `y` needs to be lowercase and `N` needs to be uppercase

Reply:
Oh ok that makes sense